src/api/app/views/layouts/webui2/webui.html.haml

```diff
@@ -9,6 +9,13 @@
     %title
       = "#{@pagetitle} - " if @pagetitle
       = home_title
+    %meta{ property: "og:title", content: "#{meta_title}"}/
+    %meta{ property: "og:type", content: 'website'}/
+    %meta{ property: "og:url", content: request.original_url }/
+    %meta{ property: "og:image", content: "#{meta_image}"}/
```

Comment:
Same here, no need to interpolate.

Reply:
I guess I went a little over board on the interpolation! Cleaned all this up.